agent/src/beerocks/slave/son_slave_thread.cpp

```diff
@@ -3258,7 +3258,7 @@ bool slave_thread::slave_fsm(bool &call_slave_select)
             break;
         }
 
-        std::copy_n(hostap_params.supported_channels, message::SUPPORTED_CHANNELS_LENGTH,
+        std::copy_n(hostap_params.preferred_channels, message::SUPPORTED_CHANNELS_LENGTH,
                     &std::get<1>(tuple_supported_channels));
```

Comment:
same here 
`preferred_channels = supported_channels` is  confusing

Reply:
Changed the all the ACS related variables to use `preferred_channels` instead